resources/tests/scripts/demonstrate_require.lua

```diff
@@ -0,0 +1,11 @@
+local demo = require "lib/demo"
```

Comment:
The goal here is to require a package installed via luarocks or somesuch?

If so, can we add a test case that covers requiring a globally installed lua dependency?